packages/valist-sdk/src/typesApi.ts

```diff
@@ -29,8 +32,10 @@ export interface ContractMetadata extends TokenMetadataInterface {
   marketplace_urls?: string[];
 }
 
-export interface ProjectMetaApi extends ProjectMetaInterface {
+export interface ProjectMetaApi extends Omit<ProjectMetaInterface, 'systemRequirements' | 'wineSupport'> {
   networks: ContractMetadata[];
+  system_requirements: SystemRequirements;
+  wine_support: WineSupport;
 }
```

Comment:
Great question, it's actually because the json payloads we anchor on the protocol use camel case (`systemRequirements`) but we're migrating to have everything match the API which uses snake case which is more standard in SQL/Postgres naming conventions (`system_requirements`).

Great catch though, these fields should probably still be optional because they can potentially be undefined.

Reply:
ah interesting. gotcha. sounds good